test/types/types.go

```diff
@@ -402,3 +403,35 @@ func GetReplicaAttachments(persistentVolume *v1.PersistentVolume, client clients
 	}
 	return azVolumeAttachmentsReplica, nil
 }
+
+func GetSchedulableNodes(azDiskClient *azDiskClientSet.Clientset, client clientset.Interface) []*v1.Node {
```

Comment:
Ideally, this function should take a `v1.Pod` and check for node taints with no matching toleration on the pod. You can use the `Toleration.ToleratesTaint()` method to check if a toleration satisfies the taint criteria.